src/commands/Social/vault.ts

```diff
@@ -61,7 +63,8 @@ export default class extends SkyraCommand {
 		const newMoney = money + coins;
 		const newVault = vault - coins;
 
-		await this.updateBalance(newMoney, newVault, settings);
+		await message.author.increaseBalance(coins, EconomyTransactionReason.Vault);
+		await settings.update(UserSettings.Vault, newVault);
```

Comment:
Convert this to one SG call, please.

Reply:
As a hint, you can restore the `updateBalance` method to not duplicate the code.